tests/src/Common/Platform/platformdefines.h

```diff
@@ -55,6 +55,11 @@ typedef const WCHAR *LPCWSTR, *PCWSTR;
 #define __stdcall
 #define _cdecl
 #endif
+#ifndef CROSS_MSC_VER_DEFINES
+#define CROSS_MSC_VER_DEFINES
+#define __FUNCSIG__ "<clang-3.5 signature not supported>"
+#define __int64 long long
+#endif // CROSS_MSC_VER_DEFINES
```

Comment:
Actually I have been trying to use intXY_t types but for some reasons it also doesn't work.